src/pyfaf/cmdline.py

```diff
@@ -92,6 +92,8 @@ def __init__(self, desc=None, prog=sys.argv[0], usage=None,
         self.add_argument("-v", "--verbose", action="store_const",
                           const=logging.DEBUG, default=logging.INFO,
                           help="turn on all verbose output except for SQL")
+        self.add_argument("-V", "--version", action="store_true",
```

Comment:
I would move this to `src/bin/faf`. Having `--version` in every command is not needed IMO.